### DISCUSSION TITLE
RGB Matrix Help

### DISCUSSION THREAD

**Discussion by bp425** [riffnshred/nhl-led-scoreboard#408] [Hardware]
Hello,

I've been following the instructions and got it set up mostly, but I've been trying to troubleshoot an issue with the screen. 

I noticed the top half of my RGB matrix was only displaying blue:https://imgur.com/a/6jj7qlE
and I ran a demo from the Github to display all the colours, here is a video with the test: https://youtu.be/3bQv7SRC9lk

I'm using a Raspberry Pi 3A+, 32x64 Adafruit Matrix and the Adafruit RGB bonnet. 
The power supply I'm using has these specs: Input: AC 100 - 240V, 50 / 60Hz ; Output: DC 5V, Max 10A/10000mA, 50W

I just wanted to see if someone can confirm if it's a matrix issue or did I mess up somewhere?

Thanks!

**Comment by riffnshred** [comment]
Check and make sure the bottom of the HAT or bonnet is not touching the raspberry pi. If that's not it, make sure the data cable is well seated.
